common/artifacts/000_wc_artifacts.txt

```diff
@@ -801,6 +1060,64 @@ dark_crystal = {
 
 	picture = "GFX_inv_misc_orb_03"
 }
+horn_of_cenarius = {
+	quality = 5
+	flags = { misc powerful_magical_artifact }
+
+	active = {
+		is_adult = yes
+		is_druid_class_trigger = yes
```

Comment:
REEE! Tyrande isn't druid! I would replace it by `is_magical_class_trigger = yes`.
https://wow.gamepedia.com/Horn_of_Cenarius#The_Third_War

Reply:
done